app.js

```diff
@@ -8,4 +58,87 @@ hamburguer.addEventListener('click', () => {
 navMenu.addEventListener('click', () => {
   hamburguer.classList.toggle('active');
   navMenu.classList.toggle('active');
-});
\ No newline at end of file
+});
+
+projects.forEach((work) => {
+  works.innerHTML += `
+    <div class="section-${work.type} main_section">
+                <div class="content_image_card">
+                    <img class="image_card" src="${work.image}" alt="snapshot tonic">
+                </div>
+                <div class="content_card">
+                    <h2 class="title_card">${work.title}</h2>
+                    <div class="subtitle_card">
+                        <h6 class="first_subtitle_card">${work.subtitle1}</h6>
+                        <img class="point_icon" src="images/point_icon.png" alt="point">
+                        <h6 class="second_subtitle_card">${work.subtitle2}</h6>
+                        <img class="point_icon" src="images/point_icon.png" alt="point">
+                        <h6 class="second_subtitle_card">${work.year}</h6>
+                    </div>
+                    <p class="paragraph_card">
+                        ${work.paragraph}
+                    </p>
+                    <div>
+                        <ul class="technologies list_technologies_${work.id}">
+                        </ul>
+                    </div>
+                    <button onClick="popupModal(${work.id})" type="button" class="button_card button"><b>See project</b></button>
+                </div>
+            </div>
+
+   `;
+  work.technologies.forEach((technology) => {
+    document.querySelector(`.list_technologies_${work.id}`).innerHTML += `<li>${technology}</li>`;
+  });
+});
+
+function popupModal(x) {
+  modal.classList.toggle('show');
+
+  modal.innerHTML += `
+                <div class="modal">
+                    <div class="modal_content">
+                        <div class="header_modal">
+                            <h2 class="title_modal">${projects[x].title}</h2>
+                            <img class="modal_close" onClick="closeModal()" src="images/close_icon.png" alt="close">
+                        </div>
+                        <div class="subtitle_card">
+                            <h6 class="first_subtitle_card">${projects[x].subtitle1}</h6>
+                            <img class="point_icon" src="images/point_icon.png" alt="point">
+                            <h6 class="second_subtitle_card">${projects[x].subtitle2}</h6>
+                            <img class="point_icon" src="images/point_icon.png" alt="point">
+                            <h6 class="second_subtitle_card">${projects[x].year}</h6>
+                        </div>
+                        <div class="content_image_modal">
+                            <img class="image_modal" src="images/snapshoot_portfolio_modal.png" alt="snapshot tonic">
+                        </div>
```

Comment:
Kindly make sure to populate the `images` dynamically and also a `pop-up` model should have the same image as the project clicked-on image.